programs/ssl/ssl_client2.c

```diff
@@ -2633,10 +2633,6 @@ int main( int argc, char *argv[] )
 
 #endif /* MBEDTLS_X509_CRT_PARSE_C */
 
-#if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL) && defined(MBEDTLS_ZERO_RTT)
-    mbedtls_ssl_conf_early_data( &conf, opt.early_data, early_data, strlen( early_data ), NULL );
```

Comment:
This should not have been moved, because we also test external PSKs, for which the early data configuration must happen prior to the _first_ handshake.